README.md

```diff
@@ -1,247 +1,260 @@
-# Centrifuge
+# Cent
 [![Code Climate](https://codeclimate.com/github/centrifugal/centrifuge-ruby/badges/gpa.svg)](https://codeclimate.com/github/centrifugal/centrifuge-ruby)
 [![Build Status](https://travis-ci.org/centrifugal/rubycent.svg)](https://travis-ci.org/centrifugal/rubycent)
 
-Ruby gem for [Centrifugo](https://github.com/centrifugal/centrifugo) real-time messaging server
+[Centrifugo HTTP API v2](https://centrifugal.github.io/centrifugo/server/http_api/) client in Ruby.
 
 ## Installation
 
 Add this line to your application's Gemfile:
 
-    gem 'centrifuge'
-
+```ruby
+gem 'cent' 
+```
 
 And then execute:
 
     $ bundle
 
 Or install it yourself as:
 
-    $ gem install centrifuge
-
-## Releases
-
-Travis CI configured to automatically release tagged commits to rubygems.
+    $ gem install cent
 
-Before tagging, please increment version in `lib/centriguge/version.rb` and use git tag like `v1.1.1`.
+## Usage
 
-Dont forget to `git push --tags`.
+Functionality is split between two classes:
+ - `Cent::Client` to call API methods
+ - `Cent::Notary` to generate tokens
 
-## Compatibility
+### Token Generation
 
-Rubycent 1.0.0+ supports only Centrifugo version 1.0.0+ with single project only.
-If you want to use multiproject Centrifugo, please use rubycent 0.1.0
+```ruby
+notary = Cent::Notary.new(secret: 'secret')
```

Comment:
Looks like only HMAC tokens supported here, maybe a better API would be sth like:

```
Cent::Token.new_hmac(secret: 'secret')
```